lib/StripeMethod.js

```diff
@@ -87,6 +87,17 @@ function stripeMethod(spec) {
       }
 
       var requestPath = this.createFullPath(commandPath, urlData);
+      var options = {headers: objectAssign(opts.headers, spec.headers)};
+
+      if (spec.validator) {
+        try {
+          spec.validator(data, options);
+        } catch (err) {
+          reject(err);
+          return;
+        }
+      }
+
```

Comment:
So, unfortunately there is no tradeoff there without more severe changes to how we do things.  The API server can't tell the difference between `Stripe-Version` passed explicitly to `stripe.ephemeralKeys.create` versus the `Stripe-Version` assigned to `stripe.setApiVersion`.  The second should be an error, as the version has to come from the frontend.